models/evc.py

```diff
@@ -1776,7 +1845,7 @@ def handle_interface_link_up(self, interface: Interface):
         }
         self.activate()
         log.info(
-            f"Activating EVC {self.id}. Interfaces: "
+            f"Activating {self.id}. Interfaces: "
```

Comment:
Any reasons to remove the EVC str? Minor stuff, but it had an impression that it was still very hand to have it, unless you meant to interpolate the `{self}` dunder repr here?

Reply:
Yes, I meant to changed it to `{self}` which displays the ID and the name.